canvas_account_admin_tools/static/canvas_account_admin_tools/js/services/ATRAPIService.js

```diff
@@ -0,0 +1,117 @@
+(function() {
+  var module = angular.module('ATRAPIModule', []);
```

Comment:
Maybe ActRestAPIModule (for Academic TechRestAPI) :-). But as discussed offline, this is a very low priority.